### PR TITLE
Events api

### PR DESCRIPTION
**Pull request by tamslo**
The missing changes for PUT and DELETE events will be on separate branches, I don't want the PR to become too big.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -51,14 +51,23 @@ CREATE TYPE alarm_action AS ENUM (
...
+-- scope_id.
 CREATE TABLE events (
   id                UUID UNIQUE PRIMARY KEY  NOT NULL DEFAULT uuid_generate_v4(),
   summary           TEXT                              DEFAULT NULL,
   location          TEXT                              DEFAULT NULL,
   description       TEXT                              DEFAULT NULL,
   dtstart           TIMESTAMP WITH TIME ZONE NOT NULL,
   dtend             TIMESTAMP WITH TIME ZONE NOT NULL,
-  scope_id      UUID                     NOT NULL,
+  scope_id          UUID                     NOT NULL,
```
Comment (MrSerth) [inline schema.sql]: ups, sorry for that mis-formatting!

File: src/routes/events.js [hunk 2]
```diff
@@ -90,36 +76,78 @@ router.put('/events/ics/:eventId', icsToJson, authorize, function (req, res) {
...
+        .then(sendUpdateNotification)
         .then(eventsToIcsInJsonApi)
         .then((jsonApi) => { returnSuccess(res, 200, jsonApi); })
         .catch((error) => { returnError(res, error); });
 });
 
+router.delete('/events/:eventId', authorize, function (req, res) {
+    const eventId = req.params.eventId;
+    // TODO delete only for scopeIds and check for alarms and exdates
+    const scopeIds = req.body.scopeIds;
```
Comment (MrSerth) [inline src/routes/events.js]: You're missing a JSON parser here, aren't you? Leave it here for now, I might have a closer look to this at the end of my current tasks.

Comment (tamslo) [inline src/routes/events.js]: As said before, the deletion I will do in another branch

Comment (MrSerth) [inline src/routes/events.js]: Great! :+1:

**Pull request merged**